### PR TITLE
[12.0][ADD] hr_expense_petty_cash_sequence

### PR DESCRIPTION
**Pull request by ps-tubtim**
This PR is waiting for https://github.com/OCA/hr/pull/621 and https://github.com/OCA/hr/pull/679, you can always use this for readability.

- [x] [12.0][MIG] hr_expense_sequence
- [x] [12.0][ADD] hr_expense_petty_cash

### CODE REVIEW

File: hr_expense_petty_cash_sequence/models/hr_expense_sheet.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, models
+
+
+class HrExpenseSheet(models.Model):
+    _inherit = 'hr.expense.sheet'
+    _rec_name = 'number'
```
Comment (kittiu) [inline hr_expense_petty_cash_sequence/models/hr_expense_sheet.py]: @ps-tubtim do we need to force this for petty cash? I think this line already exists in hr_expense_sequence.
Should we remove it?

**Review by kittiu — APPROVED** [review]
LGTM

File: hr_expense_petty_cash_sequence/models/hr_expense_sheet.py (hunk 1 above)
Comment (ps-tubtim) [reply hr_expense_petty_cash_sequence/models/hr_expense_sheet.py]: Edited.

**Review by Saran440 — APPROVED** [review]
Functional Test :+1:

**Pull request merged**